data/data.json

```diff
@@ -1,5 +1,423 @@
+  },
+  {
+    "commonName": "Flamingo Flower Plant",
+    "scientificName": "Anthurium scherzerianum",
+    "category": "Flowering plants",
+    "maxGrowth": "12 - 18 inches in height",
+    "origin": "Central and South America.",
+    "poisonousForPets": "Toxic to cats and dogs",
+    "images": {
+      "imagePrimary": "https://www.houseplantsexpert.com/image-files/flamingo-flower-plant.jpg",
+      "imageSecondary": "https://www.houseplantsexpert.com/image-files/baby_boomer1.jpg",
+      "imageTernary": ""
+    },
+    "description": "The flamingo flower plant is a species from the Anthuirum genus that displays great looking waxed effect flowers which bloom with an orange curly spadix. The Anthurium scherzerianum from the genus is suitable for a room within a home because of the acceptable height it grows up to, and of course the attractive spathe it displays for décor purposes.",
+    "temperature": "Average room temperatures from 65°F/18.3°C - 75°F/23.8°C (maybe a little higher and not below 60°F/15.5°C). Lower temperatures will affect it's leaves (may turn yellow), impede it's growth and flowering.",
+    "light": "The flamingo flower is a lover of light - but not direct sunlight.",
+    "watering": "Keeping the soil moist at all times within the summer is advisable. Water every couple of days and less in the winter.",
+    "soil": "A good draining soil is best suited, such as a peat moss based potting soil and perlite.",
+    "rePotting": "These are re-potted within spring usually every 2 years, or when they become pot-bound.",
+    "fertilizer": "",
+    "humidity": "The flamingo loves being misted with a light spray of lukewarm water, regulary. Cleaning lightly with a gentle sponge is advised, which improves humidity - and makes it look healthy.",
+    "propagation": "Propagate by dividing the plant when you decide to re-pot in spring. Having a good temperature above 75°F/23.8°C for the first few weeks will give the plant a great start in life."
+  },
+  {
+    "commonName": "Growing Moth Orchid",
+    "scientificName": "Phalaenopsis",
+    "category": "Flowering plants",
+    "maxGrowth": "Height 3ft.",
+    "origin": "Southeast Asia, Taiwan, Australia and elsewhere",
+    "poisonousForPets": "Non-toxic to cats and dogs",
+    "images": {
+      "imagePrimary": "https://www.houseplantsexpert.com/image-files/moth_orchid_white.jpg",
+      "imageSecondary": "https://www.houseplantsexpert.com/image-files/phalaenopsis1.jpg",
+      "imageTernary": "https://www.houseplantsexpert.com/image-files/phalaenopsis_pink.jpg"
+    },
+    "description": "Moth orchid is the common name for the phalaenopsis orchid that's quickly become one of the most popular species from the orchidaceae family of flowering plants. Glorious colored blooms all year round and the "ease of growing" is what makes these a popular house plant choice.",
```

Comment:
I guess strings inside of the strings are not supported in Json file. So its better to change it with ` ` instead of " ". So that we don't get warning.